src/clobber/CMakeLists.txt

```diff
@@ -25,8 +25,6 @@ target_link_libraries(clobber-lib
 )
 
 add_if_library(clobber-lib bv_refinement)
-add_if_library(clobber-lib specc)
-add_if_library(clobber-lib php)
```

Comment:
Wishlist: Could those changes (and related ones) go in a separate commit, please?